sbi/neural_nets/density_estimators/flow.py

```diff
@@ -61,3 +63,16 @@ def sample(self, sample_shape: Shape, condition: Tensor) -> Tensor:
         return self.net.sample(num_samples, context=condition).reshape(
             (*sample_shape, -1)
         )
+
+    def sample_and_log_prob(
```

Comment:
for `nflows`, the `sample_and_log_prob` method is already implemented, so we can just call it here.

Reply:
Oh, great will change it.